test/torch/tensors/test_additive_shared.py

```diff
@@ -180,6 +180,24 @@ def test_mul(workers):
     assert (z == (t * t)).all()
 
 
+def test_operate_with_constants(workers):
+    bob, alice, james = (workers["bob"], workers["alice"], workers["james"])
+    x = th.tensor([1.0])
+    x_sh = x.fix_precision().share(alice, bob, crypto_provider=james)
+
+    r_sh = x_sh + 0
+    assert r_sh.get().float_prec() == x
```

Comment:
Could you modify the test so that the result is not equal to x? Otherwise the test won't distinguish between correct summation/multiplication and not doing anything.

Reply:
I've tried to, and I'm getting the following assertion error:

```python
def test_operate_with_constants(workers):
    bob, alice, james = (workers["bob"], workers["alice"], workers["james"])
    x = th.tensor([1.0])
    x_sh = x.fix_precision().share(alice, bob, crypto_provider=james)

    r_sh = x_sh + 10
    assert r_sh.get().float_prec() == x + 10

    r_sh = x_sh - 7
    assert r_sh.get().float_prec() == x - 7

    r_sh = x_sh * 2
    assert r_sh.get().float_prec() == x * 2

    r_sh = x_sh / 0.5
    assert r_sh.get().float_prec() == x / 0.5
```

Error:

```
 assert r_sh.get().float_prec() == x + 10
E       assert tensor([1.0100]) == tensor([11.])
```

Reply:
Thanks for testing this! I will limit myself to integers because as mentioned in another thread operating fixed_precision with float values should not be possible: float should be converted to fixed_precision explicetly (at least for the moment until we are confident enough with what we're doing)

Reply:
Nice, thanks @LaRiffle !